### ISSUE TITLE
Option to disable zoom

### ISSUE THREAD

**Issue by vpekarek** [lucaspulliese/vue-cool-lightbox#99]
I know that the zoom is disabled by default for mobile devices. 
But would be nice to have an option to set this. I would like to disable zooming for everything, but now I can't.

**Comment by jakubmirejovsky** [comment]
On the other hand, option for "always zoom" would be great as well!

**Comment by oriongunning** [comment]
yes that would be great. zooming on mobile is essential. thanks for a great plugin though :)
